index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="icon" type="image/png" sizes="32x32" href="/Images-Ecom/rr-favicon-2.png">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="https://robyn-ui.netlify.app/components/import.css">
+    <link rel="stylesheet" href="/index.css">
+    <title>Robyn Reads | Home</title>
+</head>
+<body>
+    <header class="rr-nav-container">
+        <div class="header-logo"><a href="#"><img class="logo" src="/Images-Ecom/rr-logo.png"></a></div>
+        
+        <nav class="rr-nav-box">
+            <input class="nav-search" type="search" placeholder="Search">
+            <div class="nav-item"><button class="btn btn-float nav-btn nav-btn-icon" onclick="window.location.href='/Screens/Authentication';"><i class="fa fa-sign-in" aria-hidden="true"></i>
+            </button></div>
+            <div class="nav-item"><button class="btn btn-float nav-btn nav-btn-icon" onclick="window.location.href='/Screens/Authentication';"><i class="fa fa-heart" aria-hidden="true"></i>
+            </button></div>
+            <div class="nav-item"><button class="btn btn-float nav-btn nav-btn-icon" onclick="window.location.href='/Screens/Authentication';"><i class="fa fa-shopping-cart" aria-hidden="true"></i>
+            </button></div>
+        </nav>
+    </header>
+
+    <main  class="landing-img-container">
+    
+            <div class="background-img-wrapper">
+                <img class="image-responsive rr-image-responsive" src="/Images-Ecom/rr-landing-background.jpg" alt='Landing Background Image'/>
+            </div>
+            <div class="landing-title-wrapper">
+                <h1>Welcome to Robyn Reads</h1>
+                <h3>Explore the World of Comics Here.</h3>
+                <button class="btn btn-primary landing-btn" onclick="window.location.href='/Screens/Product-Listing/productlisting.html';" >Explore Now</button>
```

Comment:
Try avoiding inline JS

Reply:
That is done temporarily. I had planned to remove that while implementing functionality. Thank you for pointing it out.